service/tests/test_load.py

```
@@ -0,0 +1,49 @@
+    person = persons[0]
+    persons = [person]
+
+    statement = select(Person).where(Person.id == person["uuid"])
+    assert session.execute(statement).first() is None
+
+    lf = app.load.pure.persons.transform_persons(persons)
+    df = lf.collect_batches(chunk_size=1)
+
+    print("Created persons iterator")
+
+    for df in df:
```

Ruff (0.15.6)

[warning] Loop control variable `df` overrides iterable it iterates

(B020)